src/activemasternode.cpp

```diff
@@ -30,7 +32,15 @@ void CActiveMasternode::ManageStatus()
         pmn = mnodeman.Find(pubKeyMasternode);
         if(pmn != NULL) {
             pmn->Check();
-            if(pmn->IsEnabled() && pmn->protocolVersion == PROTOCOL_VERSION) EnableHotColdMasterNode(pmn->vin, pmn->addr);
+            if(pmn->IsEnabled()) {
+                if(pmn->protocolVersion == PROTOCOL_VERSION)
+                    EnableHotColdMasterNode(pmn->vin, pmn->addr);
+                else {
+                    status = ACTIVE_MASTERNODE_DIFF_PROTO;
+                    LogPrintf("CActiveMasternode::ManageStatus() - %s\n", GetStatus());
+                    return;
```

Comment:
Shouldn't we allow the masternode to broadcast a new message and update the network? The way this is written, the node operator would be required to stop the masternode first before starting the new one, right?